maintenance/README.md

```diff
@@ -6,35 +6,63 @@ This project contains a collection of scripts to use a maintenance container to
 
 To set up a local development environment, run these commands from anywhere in repo:
 
- 1. Build and run the containers: `docker-compose up --detach --build`
- 2. To setup the apps: `docker-compose run --rm maintenance bash setup.sh`
-     - <details>
-       <summary>If you encounter unexpected hangs without error during <code>npm install</code>...</summary>
-       
-       ...try stopping `mock-auth` container temporarily. (It runs a Java app,
-       and these can be memory hogs.) You can do this via Docker UI or this CLI
-       command:
-       ```
-       docker-compose stop mock-auth
-       # Finish building app.
-       docker-compose start mock-auth
-       ```
-       </details>
- 3. Open a browser and navigate to http://localhost:8000/login and login in
-    with `admin@test.com` as "user/subject": (the "claims" text box can be left
-    blank, and there is no password)
- 4. You should be automatically redirected to http://localhost:8000/admin
+1. Build and run the containers: `docker-compose up --detach --build`
+2. To setup the apps: `docker-compose run --rm maintenance bash setup.sh`
+   - <details>
+        <summary>If you encounter unexpected hangs without error during <code>npm install</code>...</summary>
+        ...try stopping `mock-auth` container temporarily. (It runs a Java app,
+        and these can be memory hogs.) You can do this via Docker UI or this CLI
+        command:
+        ```
+        docker-compose stop mock-auth
+        # Finish building app.
+        docker-compose start mock-auth
+        ```
+     </details>
+3. Next you can log in:
+   - For testing admin accounts:
+     1. Navigate to http://localhost:8000/login
+     2. Enter `admin@test.com` as the "User/subject" (the "Claims" input can be left blank, and there is no password)
+     3. You should be redirected to http://localhost:8000/admin/dashboard
```

Comment:
I think this changed since you started your branch.  /login and /login-info should redirect you to your profile, now.

Reply:
Thanks, I'll update it

Reply:
for hoisting to issue todo list

- [ ] fix `maintenance/README.md` regressions (https://github.com/GCTC-NTGC/gc-digital-talent/pull/3051#discussion_r949373506)